CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master
 
+## 0.5.0 (2018-08-29)
+
+- [PR [#19](https://github.com/palkan/isolator/pull/19)] Adding support for ruby version 2.2.2. ([@shivanshgaur][])
```

Comment:
Please, add you handle link to the end of the file to make it work

Reply:
Done